core/src/global/_utility-classes.scss

```
@@ -380,3 +380,40 @@
 .tds-u-overflow-hidden {
   overflow: hidden !important;
 }
+
+// Flex
+
+.tds-u-flex-start {
+  display: flex !important;
+  justify-content: flex-start !important;
+}
+
+.tds-u-flex-center {
+  justify-content: space-between !important;
+  align-items: center !important;
+}
+
+.tds-u-flex-end {
+  display: flex !important;
+  justify-content: flex-end !important;
+}
+
+.tds-u-flex-dir-col {
+  flex-direction: column !important;
+}
+
+.tds-u-gap1 {
+  gap: 8px !important;
+}
+
+.tds-u-gap2 {
+  gap: 16px !important;
+}
+
+.tds-u-flex-gap3 {
+  gap: 24px !important;
+}
```

```suggestion
.tds-u-gap3 {
  gap: 24px !important;
}
```

To match other classes